wasm_crypto/index.html

```diff
@@ -0,0 +1,85 @@
+<html>
```

Comment:
More like a point to discuss, but maybe it would be better to create some kind of `crypto_binding.js` instead of a web page. Then init and call required functions there and execute .js file from console. This way we can later integrate it into our CI and maintain them easier, considering that number of bindings will grow.